vuln/npm/407.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": 407,
+  "title": "deap prototype pollution",
+  "overview": "deap node module before 1.0.1 suffers from a prototype pollution vulnerability",
+  "created_at": "2018-02-17",
+  "updated_at": "2018-04-09",
+  "publish_date": "2018-04-09",
+  "author": "Olivier Arteau (HoLyVieR)",
+  "module_name": "deap",
+  "cves": [],
+  "vulnerable_versions": "<1.0.1",
```

Comment:
<= , no?

Reply:
My bad here; 1.0.1 is safe so it should get to patched_versions

Reply:
👍